### PR TITLE
ARROW-11122: [Rust] Added FFI support for date and time.

### PR DESCRIPTION
**Pull request by jorgecarleitao**
Like the PR title says, it is just completing the matrix.

### CODE REVIEW

File: rust/arrow-pyarrow-integration-testing/src/lib.rs [hunk 1]
```diff
@@ -153,10 +153,25 @@ fn substring(array: PyObject, start: i64, py: Python) -> PyResult<PyObject> {
     to_py(array, py)
 }
 
+/// Returns the concatenate
```
Comment (nevi-me) [inline rust/arrow-pyarrow-integration-testing/src/lib.rs:156]: Is the doc comment complete?

**Review by nevi-me — APPROVED** [review]
LGTM

File: rust/arrow-pyarrow-integration-testing/src/lib.rs (hunk 1 above)
Comment (alamb) [reply rust/arrow-pyarrow-integration-testing/src/lib.rs:156]: Looks like it to me -- but I think it can be improved in a follow on PR

**Review by alamb — APPROVED** [review]
I took a quick read and I also think it looks good to me.

**Pull request closed without merging**